static/timeago.js

```diff
@@ -0,0 +1,20 @@
+var timestamp = document.getElementById('time').getAttribute('data-time');
```

Comment:
put all of this into two function.

The first function should be a HTML unaware utility function which takes a `timestamp` and `ago` value, and returns the timestring.

The function second should be a function which is given two HTML element objects, one for the `time` and the other for `ago`, and this section function would call the first function to get the computed value and would put that value into the `ago` element.

Optionally add a third function which takes no arguments, fetches the desired elements, and calls the second function.

The second or third function should be invoked from inside the HTML ; worse case, use `body.onload` to invoke it.